projects/observability/src/shared/components/cartesian/cartesian-chart.component.ts

```diff
@@ -93,8 +93,8 @@ export class CartesianChartComponent<TData> implements OnChanges, OnDestroy {
         )
       );
 
-    if (this.range) {
-      this.chart.withRanges(this.range);
+    if (this.ranges !== undefined) {
```

Comment:
better to check this via truthiness explicitly (since this could be null if provided via an async pipe, even though we don't represent it in the type)